src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -916,6 +915,17 @@ public static FormulaValue AbsDecimal(IRContext irContext, DecimalValue arg)
             return new DecimalValue(irContext, val);
         }
 
+        public static FormulaValue Trunc(IRContext irContext, FormulaValue[] args)

```

Comment:
>Trunc [](http://example.com/codeflow?start=35&length=5)

In case we need to change `RoundDown`, `Trunc` function will have its own library function to skip some conditions.